modules/admin-portal/pages/monitoring/log-viewer.adoc

```diff
@@ -57,4 +57,33 @@ image:single-search-result.png[image]
 
 Clicking on a single search result will show the log content in the display panel and highlight the matching log line.
 
-image:line-highlight-log (1).png[image]
\ No newline at end of file
+image:line-highlight-log (1).png[image]
+
+== Read-Only Access to Logs
+
+To streamline the debugging and optimization process, *read-only access to logs* has been introduced for application developers. This feature allows developers to independently view and monitor logs through *GraphStudio* or the *TigerGraph Cloud Portal* without requiring sysadmin intervention.
```

Comment:
"has been introduced": when?  Remember that we have a standard sentence structure for taking about "new" features: "As of X.X, users other than `admin` users can be granted the privilege to read logs."

"independently" : independent from what? Does this word add anything to the sentence? If I remove it, the sentence still seems to say all that is important.

"sysadmin": This is a slang abbreviation. Do not use it. Use the full phrase.